os/src/Path.scala

```diff
@@ -43,9 +43,9 @@ object PathChunk {
   implicit class SeqPathChunk[T](a: Seq[T])(implicit f: T => PathChunk) extends PathChunk {
     var segments0 = Nil
     var ups0 = 0
```

Comment:
Any idea, why these are here? Could you add some `@deprecated` on them, so we can remove them in the next major bump?